src/Components/Dropdowns/Dropdown.scss

```diff
@@ -304,7 +304,7 @@ $dropdown-item--padding-v: $cf-space-2xs;
   }
 
   .cf-dropdown-item.active:not(.cf-dropdown-item__disabled) {
-    background-color: $cf-martinique;
+    background-color: #2A2649;
```

Comment:
Do we not have a color variable for this color?

Reply:
Design team will be sending it to me after this week, basically, i'll be going through and deleting old colors and adding new ones with their names, and I thought I can just address this then (i'll be doing a global hex search to replace any hard coded values)

Reply:
Gotcha!